src/server/qgsserver.cpp

```diff
@@ -81,6 +80,13 @@ QgsServer::~QgsServer()
 }
 
 
+QString& QgsServer::serverName()
```

Comment:
No need to return a reference here. Qstrings are implicitly shared, so returning a copy is a near no cost operation